apps/teamsjs-cc-app/tabs/src/components/capabilities/Dialog.AdaptiveCard.tsx

```diff
@@ -0,0 +1,45 @@
+import * as adaptiveCardJsonData from "./data/dialog.adaptivecard.json";
```

Comment:
I don't see a file that is testing Dialog.AdaptiveCard.Bot

Reply:
This file only contains adaptive card example data, so it can be used in both capabilities, Dialog.AdaptiveCard and Dialog.AdaptiveCard.Bot.